library/src/main/java/com/novoda/downloadmanager/BatchFileBuilder.java

```diff
@@ -0,0 +1,46 @@
+package com.novoda.downloadmanager;
+
+/**
+ * Adds {@link BatchFile} to the parent {@link Batch} using the {@link BatchBuilder}
+ * and {@link BatchFileBuilder} fluent APIs.
+ */
+public interface BatchFileBuilder {
+
+    /**
+     * Sets {@link BatchFileBuilder} to build a {@link BatchFile} with a corresponding
+     * {@param downloadFileId} that can be used to lookup a download file belonging to
```

Comment:
What about replacing 
`that can be used to lookup a download file`
with
`that can be later used to retrieve a downloaded file`?